docs/upgrade-guide.md

```diff
@@ -2,6 +2,10 @@
 
 ## Upgrading from deck.gl v8.3 to v8.4
 
+### webglcontextlost
+
+- onError callback is called in case of webglcontextlost event. You can provide additional information to the user about what happened. Rendering is stopped until webglcontextrestored event.
```

Comment:
The upgrade guide only lists deprecations and breaking changes. Remove if there is no clear actions on the user's behalf.